src/operator-question-mark.bs

```diff
@@ -40,28 +40,27 @@ The following is an example of how the SerenityOS Kernel networking stack [creat
 where its contract is to return a kernel error enum value, or a value of the template parameter `T`.
```

Comment:
Make sure to also update the perma-link so it will also contain the relevant changes (see line above)

Reply:
Oh great point! Fixed.